Do not build usbredirserver. It is deprecated, has been removed from
the upstream code already, and cannot build on OS X 10.8 and earlier.
https://gitlab.freedesktop.org/spice/usbredir/-/commit/f4ffdce329305da2803684776f7659083a530819
--- configure.ac.orig	2021-04-02 10:51:59.000000000 -0500
+++ configure.ac	2022-04-05 19:48:28.000000000 -0500
@@ -123,7 +123,6 @@
 usbredirhost/libusbredirhost.pc
 usbredirparser/Makefile
 usbredirparser/libusbredirparser-0.5.pc
-usbredirserver/Makefile
 usbredirtestclient/Makefile
 fuzzing/Makefile
 ])
--- Makefile.am.orig	2021-04-02 10:51:59.000000000 -0500
+++ Makefile.am	2022-04-05 19:48:15.000000000 -0500
@@ -1,6 +1,6 @@
 SUBDIRS = usbredirparser usbredirhost docs fuzzing tools
 if ! OS_WIN32
-SUBDIRS += usbredirserver  usbredirtestclient
+SUBDIRS += usbredirtestclient
 endif
 
 EXTRA_DIST =			\
